package.json

```diff
@@ -59,26 +59,28 @@
     "babel-eslint": "10.1.0",
     "eslint": "7.22.0",
     "eslint-config-airbnb-base": "14.2.1",
+    "eslint-config-klopov": "^1.0.2",
```

Comment:
We can remove this as it's not being used now, correct?

Reply:
correct